controllers/mysqlcluster_controller.go

```diff
@@ -942,7 +942,23 @@ func (r *MySQLClusterReconciler) reconcileV1PDB(ctx context.Context, req ctrl.Re
 		return client.IgnoreNotFound(err)
 	}
 
-	maxUnavailable := intstr.FromInt(int(cluster.Spec.Replicas / 2))
+	backupCronJobIsRnning := false
+	// check if backup cronjob is running
+	if cluster.Spec.BackupPolicyName != nil {
+		cj := &batchv1.CronJob{}
+		if err := r.Get(ctx, client.ObjectKey{Namespace: cluster.Namespace, Name: cluster.BackupCronJobName()}, cj); err == nil {
+			if cj.Status.Active != nil {
+				backupCronJobIsRnning = true
+			}
+		}
+	}
+
+	var maxUnavailable intstr.IntOrString
+	if backupCronJobIsRnning {
+		maxUnavailable = intstr.FromInt(int(cluster.Spec.Replicas))
```

Comment:
maxUnavailable means the maximum allowed **unavailable** Pods.
If it's set to the same as `spec.replicas`, all Pods can be gone.

Reply:
@ymmt2005 
Thank you, I misunderstood the behavior of maxUnavailable.
I fixed it so that maxUnavailable equals 0 when backupCronJOB is executed.
The title of the PR has been corrected at the same time.

4479fcceddb674c2327e48377ec4e4cb3bfb1517

Reply:
Sorry. I didn't correct the test case enough.
82227d0b952aef2e4635b0a6b8c188ddba06d52b